libmscore/beam.cpp

```diff
@@ -388,9 +388,12 @@ void Beam::layout1()
                         _up = false;
                   else if (c1) {
                         Measure* m = c1->measure();
+                        Fraction totalTicks = Fraction(0, 1);
+                        for (int i = 0; i < _elements.size(); ++i)
```

Comment:
isn't `_elements.size()`returnine a `size_t`? If so, `i` should be too

Reply:
Changed.